other/bootstrap_daemon/README.md

```diff
@@ -1,33 +1,30 @@
-##Instructions for Debian
+##Instructions 
 
 For security reasons we run the daemon under its own user.
 Create a new user by executing the following:
 ```sh
-sudo useradd --system --shell /sbin/nologin --comment "Account to run Tox's DHT bootstrap daemon" --user-group tox-bootstrapd
+sudo useradd --home-dir /var/lib/tox-bootstrapd --create-home --system --shell /sbin/nologin --comment "Account to run Tox's DHT bootstrap daemon" --user-group tox-bootstrapd
 ```
 
-Create a directory where the daemon will store its keys:
-```sh
-sudo mkdir /var/lib/tox-bootstrapd/
-```
+Place `tox-bootstrapd.conf` file to where `CFGFILE` variable from `tox-bootstrapd.sh` tells (for `init.d` users) or `ExecStart=` from `tox-bootstrap.service` ( for `systemd` users). By default it's `/etc/tox-bootstrapd.conf`.
+
+Go over everything in `tox-bootstrapd.conf`. Make sure `pid_file_path` matches `PIDFILE` from `tox-bootstrapd.sh` (`init.d`) or `PIDFile=` from `tox-bootstrap.service` AND file in `ExecStartPre`(`systemd`).
 
-Restrain other users from accessing the directory:
+
+Restrict access to home directory:
 ```sh
-sudo chown tox-bootstrapd:tox-bootstrapd /var/lib/tox-bootstrapd/
-sudo chmod 700 /var/lib/tox-bootstrapd/
+sudo chmod 700 /var/lib/tox-bootstrapd
 ```
 
-Look at the variable declarations in the beginning of `tox-bootstrapd.sh` init script to see if you need to change anything for it to work for you. The default values must be fine for most users and we assume that you use those next.
-
-Go over everything in `tox-bootstrapd.conf`. Make sure `pid_file_path` matches `PIDFILE` from `tox-bootstrapd.sh`.
+##For `init.d` users:
 
-Place `tox-bootstrapd.conf` file to where `CFGFILE` variable from `tox-bootstrapd.sh` tells. By default it's `/etc/tox-bootstrapd.conf`.
+Look at the variable declarations in the beginning of `tox-bootstrapd.sh` init script to see if you need to change anything for it to work for you. The default values must be fine for most users and we assume that you use those next.
 
 Place `tox-bootstrapd.sh` init file at `/etc/init.d/tox-bootstrapd` (note the disappearance of ".sh" ending).
 
 Set permissions for the init system to run the script:
 ```sh
-sudo chmod 755 /etc/init.d/tox-bootstrapd
+sudo mkdir 755 /etc/init.d/tox-bootstrapd
```

Comment:
The change from `chmod` to `mkdir` is wrong. `/etc/init.d/tox-bootstrapd` is not a directory.

>Place `tox-bootstrapd.sh` init file at `/etc/init.d/tox-bootstrapd` (note the disappearance of ".sh" ending).

I meant 
```sh
mv tox-bootstrapd.sh /etc/init.d/tox-bootstrapd
```